vendor/CMakeLists.txt

```diff
@@ -72,3 +72,7 @@ install(TARGETS adb fastboot "${ANDROID_MKE2FS_NAME}"
 set(BASH_COMPLETION_DIR "${CMAKE_INSTALL_FULL_DATADIR}/bash-completion/completions")
 install(FILES adb/adb.bash RENAME adb DESTINATION "${BASH_COMPLETION_DIR}")
 install(FILES core/fastboot/fastboot.bash RENAME fastboot DESTINATION "${BASH_COMPLETION_DIR}")
+
+# Install license files.
+set(LICENSES "${CMAKE_INSTALL_FULL_DATADIR}/licenses/android-tools")
```

Comment:
Just one more minor nitpicky thing: I would suggest renaming this variable to `LICENSE_DIR`, these variables can be overwritten externally using `cmake -D<var name>`. Adding the `_DIR` postfix makes its purpose more clear and is also consistent with the `BASH_COMPLETION_DIR` variable. Otherwise LGMT.

Reply:
> Just one more minor nitpicky thing: I would suggest renaming this variable to `LICENSE_DIR`, these variables can be overwritten externally using `cmake -D<var name>`. Adding the `_DIR` postfix makes its purpose more clear and is also consistent with the `BASH_COMPLETION_DIR` variable. Otherwise LGMT.

Done. Thanks for the suggestion.